pkg/scheduler/core/generic_scheduler.go

```diff
@@ -555,6 +554,7 @@ func (g *genericScheduler) findNodesThatFit(ctx context.Context, state *framewor
 			if !extender.IsInterested(pod) {
 				continue
 			}
+			// TODO(Huang-Wei): refactor the signature of extender Filter() and have it UTed.
```

Comment:
It's extender's Filter(), not the generic framework.Filter():

https://github.com/kubernetes/kubernetes/blob/a8caae554fb1812855a0b55726f4a38afcb0ec82/pkg/scheduler/algorithm/scheduler_interface.go#L35-L37

It should be adherent with:

https://github.com/kubernetes/kubernetes/blob/a8caae554fb1812855a0b55726f4a38afcb0ec82/pkg/scheduler/framework/v1alpha1/interface.go#L261

Am I correct?

Reply:
I don't think so, we will call the extender with all filtered nodes (make a single call, not one per node), otherwise it will be very expensive. Also, this is a v1 API, good luck changing that :)

Reply:
lol... Let me see how it goes.